### PR TITLE
[201911][docker-teamd]: Increase teammgrd timeout to allow graceful shutdown

### LINKED ISSUE #7591: [201911] PortChannel IP addresses are lost after 'config reload'
#### Description
After running 'config reload', the assigned IP addresses on LAG interfaces are lost.

#### Steps to reproduce the issue:
1. Install a fresh image based on 201911 branch.
2. Configure PortChannels on the switch.
3. Run 'config reload' command and observe IP address is missing.

#### Describe the results you received:
In the log we can see an ERROR for assigning the IP to the PortChannel from intfmgrd:

**Apr 22 15:31:25.304056 t1s9015-d INFO swss#supervisord: intfmgrd RTNETLINK answers: File exists
Apr 22 15:31:25.304756 t1s9015-d ERR swss#intfmgrd: :- setIntfIp: Command ‘/sbin/ip address “add” “10.210.1.57/31” dev “PortChannel9”’ failed with rc 2**

This means the de-init flow didn't remove the ip address of the PortChannel from the kernel.
Starting the flow again will recreate the LAG interfaces and update STATE DB to return 'true' for 'isIntfStateOk(alias)' and then run 'setIntfIp(alias, "add", ip_prefix)' which failing because the IP is already configured on the kernel.
Eventually this IP will be deleted by another process I am not sure which one, but the IP address is missing and the PortChannel exist.
To prove this claim, I stopped all services manually without starting them again and the IP addresses configured to some PortChannels were still exist.
To stop the services related to LAGS I issued "systemctl stop swss.service" which will stop 'teamd.service' as well as it is a dependent service.

**After stopping the services:**

root@Spine4700-5:/home/admin# docker ps
CONTAINER ID        IMAGE                                COMMAND                  CREATED             STATUS                       PORTS               NAMES
25e1bdb4a417        docker-snmp-sv2:latest               "/usr/bin/supervisord"   10 minutes ago      Exited (137) 6 minutes ago                       snmp
2d6ec9fff300        docker-sonic-mgmt-framework:latest   "/usr/bin/supervisord"   10 minutes ago      Exited (0) 6 minutes ago                         mgmt-framework
7ee5967b3f8f        docker-sonic-telemetry:latest        "/usr/bin/supervisord"   10 minutes ago      Exited (0) 6 minutes ago                         telemetry
ca7cb7a27cf9        docker-platform-monitor:latest       "/usr/bin/docker_ini…"   13 minutes ago      Exited (0) 5 minutes ago                         pmon
ff427570950e        docker-dhcp-relay:latest             "/usr/bin/docker_ini…"   13 minutes ago      Exited (0) 5 minutes ago                         dhcp_relay
55eeb8cfd1a9        docker-router-advertiser:latest      "/usr/bin/docker-ini…"   13 minutes ago      Exited (0) 5 minutes ago                         radv
d47b0a58a493        docker-lldp-sv2:latest               "/usr/bin/docker-lld…"   13 minutes ago      Up 10 minutes                                    lldp
3a807715a8ed        docker-syncd-mlnx:latest             "/usr/bin/supervisord"   13 minutes ago      Exited (0) 5 minutes ago                         syncd
58416e6f5b4a        docker-teamd:latest                  "/usr/bin/supervisord"   13 minutes ago      Exited (0) 5 minutes ago                         teamd
e7b9495252e0        docker-orchagent:latest              "/usr/bin/docker-ini…"   13 minutes ago      Exited (0) 6 minutes ago                         swss
5d14bea8a0ac        docker-fpm-frr:latest                "/usr/bin/supervisord"   14 minutes ago      Up 10 minutes                                    bgp
0b4ead33d921        docker-database:latest               "/usr/local/bin/dock…"   14 minutes ago      Up 14 minutes                                    database

root@Spine4700-5:/home/admin# ifconfig
Bridge: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet6 fe80::fc6b:25ff:fe3c:5177  prefixlen 64  scopeid 0x20<link>
        ether 7e:b0:bb:84:e6:5d  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 2  bytes 260 (260.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

**PortChannel8**: flags=4099<UP,BROADCAST,MULTICAST>  mtu 9178
        inet 10.210.2.203  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 2001:558:1b0:8182::2cb  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

**PortChannel9**: flags=4099<UP,BROADCAST,MULTICAST>  mtu 9178
        inet 10.210.3.9  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 2001:558:1b0:8182::309  prefixlen 127  scopeid 0x0<global>
        inet6 2001:558:1b0:8182::309  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

**PortChannel73**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.112  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::1170  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 63  bytes 4086 (3.9 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 13 overruns 0  carrier 0  collisions 0

**PortChannel74**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.114  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::1172  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 63  bytes 3978 (3.8 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0

**PortChannel75**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.116  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::1174  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 58  bytes 3748 (3.6 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 24  bytes 1748 (1.7 KiB)
        TX errors 0  dropped 16 overruns 0  carrier 0  collisions 0

**PortChannel76**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.118  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 2001:558:1b0:8182::1176  prefixlen 127  scopeid 0x0<global>
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 60  bytes 3840 (3.7 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0

**PortChannel77**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.120  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 2001:558:1b0:8182::1178  prefixlen 127  scopeid 0x0<global>
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 62  bytes 4040 (3.9 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 11 overruns 0  carrier 0  collisions 0

**PortChannel78**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.122  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::117a  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 60  bytes 3840 (3.7 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0

**PortChannel79**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.124  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::117c  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 60  bytes 3840 (3.7 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0

**PortChannel80**: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 9178
        inet 10.210.17.126  netmask 255.255.255.254  broadcast 0.0.0.0
        inet6 fe80::1eaa:aaff:feaa:aa00  prefixlen 64  scopeid 0x20<link>
        inet6 2001:558:1b0:8182::117e  prefixlen 127  scopeid 0x0<global>
        ether 1c:aa:aa:aa:aa:00  txqueuelen 1000  (Ethernet)
        RX packets 60  bytes 3840 (3.7 KiB)
        RX errors 0  dropped 9  overruns 0  frame 0
        TX packets 31  bytes 2130 (2.0 KiB)
        TX errors 0  dropped 9 overruns 0  carrier 0  collisions 0

docker0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        inet 240.127.1.1  netmask 255.255.255.0  broadcast 240.127.1.255
        inet6 fe80::1  prefixlen 64  scopeid 0x20<link>
        inet6 fd00::1  prefixlen 80  scopeid 0x0<global>
        ether 02:42:fd:5b:f9:9c  txqueuelen 0  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

eth0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 11.11.11.105  netmask 255.255.255.0  broadcast 11.11.11.255
        inet6 fe80::e42:a1ff:fecf:417a  prefixlen 64  scopeid 0x20<link>
        ether 0c:42:a1:cf:41:7a  txqueuelen 1000  (Ethernet)
        RX packets 3399  bytes 407357 (397.8 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 275865  bytes 44155824 (42.1 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
        device memory 0xf8100000-f81fffff

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        inet6 ::1  prefixlen 128  scopeid 0x10<host>
        loop  txqueuelen 1  (Local Loopback)
        RX packets 469163  bytes 70773004 (67.4 MiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 469163  bytes 70773004 (67.4 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

#### Describe the results you expected:
All IP addresses should be assigned successfully and noe ERR messages should be raised.

#### Output of `show techsupport`:
[sonic_dump_Spine4700-5_20210429_145731.zip](https://github.com/Azure/sonic-buildimage/files/6465852/sonic_dump_Spine4700-5_20210429_145731.zip)

#### Additional information you deem important (e.g. issue happens only occasionally):
This issue is not reproduced ~20% of the time which indicate some race condition between processes.

### PR DESCRIPTION
**Pull request by nazariig**
Signed-off-by: Nazarii Hnydyn <nazariig@nvidia.com>

Adding a missing part of https://github.com/Azure/sonic-buildimage/pull/6537

Resolves  #7591 

#### Why I did it
* To fix LAG IP configuration race

#### How I did it
* Extended timeout for teammgrd

#### How to verify it
1. Add >80 router LAGs
2. Do config reload

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [X] 201911
- [ ] 202006
- [ ] 202012

#### Description for the changelog

* N/A

#### A picture of a cute animal (not mandatory but encouraged)
```
      .---.        .-----------
     /     \  __  /    ------
    / /     \(  )/    -----
   //////   ' \/ `   ---
  //// / // :    : ---
 // /   /  /`    '--
//          //..\\
       ====UU====UU====
           '//||\\`
             ''``
```

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@nazariig can you please explain why this is needed and what is the system behaviour if it is not available?
@lguohan is it by purpose that this PR has no build check?

**Review by liat-grozovik — APPROVED** [review]

**Comment by lguohan** [comment]
i am not aware of. @xumia, @liushilongbuaa , can you check why pr build is not triggered?

File: dockers/docker-teamd/supervisord.conf [hunk 1]
```diff
@@ -44,6 +44,7 @@ command=/usr/bin/teammgrd
 priority=3
 autostart=false
 autorestart=false
+stopwaitsecs=60
```
Comment (judyjoseph) [inline dockers/docker-teamd/supervisord.conf:47]: Do you see teamsyncd taking longer time on this testbed ? if that is the case we can add this setting "stopwaitsecs" for teamsyncd as well.

**Comment by prsunny** [comment]
Discussed with Judy, and request to add teamsyncd changes as a later PR.

File: dockers/docker-teamd/supervisord.conf (hunk 1 above)
Comment (nazariig) [reply dockers/docker-teamd/supervisord.conf:47]: @judyjoseph i did some investigation regarding that and it seems that we don't need to increase the timeout for `teamsyncd`:

```bash
root@sonic:/# supervisorctl start teamsyncd

root@sonic:/home/admin# cat /var/log/syslog | grep teamsyncd
May 27 16:51:24.050896 sonic INFO teamd#supervisord 2021-05-27 16:51:24,050 INFO spawned: 'teamsyncd' with pid 370
May 27 16:51:24.060687 sonic NOTICE teamd#teamsyncd: :- loadRedisScript: lua script loaded, sha: 88270a7c5c90583e56425aca8af8a4b8c39fe757
May 27 16:51:24.065430 sonic INFO teamd#supervisord: teamsyncd Listens to teamd events...
May 27 16:51:24.172000 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.200503 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list:
May 27 16:51:24.200995 sonic INFO teamd#supervisord: teamsyncd get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.376369 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.411137 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.438195 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.470041 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:24.957463 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:25.051005 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:25.200879 sonic INFO teamd#supervisord: teamsyncd libteam: get_ifinfo_list: get_ifinfo_list: nl_recvmsgs failed
May 27 16:51:29.396578 sonic INFO teamd#supervisord 2021-05-27 16:51:29,396 INFO success: teamsyncd entered RUNNING state, process has stayed up for > than 5 seconds (startsecs)
```

**diff:** May 27 16:51:29.396578 - May 27 16:51:24.050896 => 5 sec

```bash
root@sonic:/# supervisorctl stop teamsyncd

May 27 16:52:48.727690 sonic INFO teamd#supervisord 2021-05-27 16:52:48,727 INFO waiting for teamsyncd to stop
May 27 16:52:49.728149 sonic NOTICE teamd#teamsyncd: :- cleanTeamSync: Cleaning up LAG teamd resources ...
May 27 16:52:49.765961 sonic NOTICE teamd#teamsyncd: :- main: Received SIGTERM Exiting
May 27 16:52:49.781918 sonic INFO teamd#supervisord 2021-05-27 16:52:49,781 INFO stopped: teamsyncd (exit status 0)
```

**diff:** May 27 16:52:49.781918 - May 27 16:52:48.727690 => 1 sec

**Pull request merged**